templates/less/components/form.less

```diff
@@ -1,20 +1,4 @@
-.formContainer {
-  width: 290px;
-  height: 280px;
-  margin: 0 auto;
-  @media (min-width : 360px) and
-  (max-width : 450px){
-    width: 320px;
-  }
-  @media (min-width : 451px) {
-    width: 400px;
-    height: 250px;
-  }
-  border-radius: 7px;
-  padding-top: 25px;
-  margin-bottom: 30px;
-  background-color: white;
-
+form {
```

Comment:
yeah: don't use a form unless it's *actually* a form. We're not using real forms, so let's not use a tag that adds lots of semantic interpretation. We already have `div[role=form]` to select with